### PR TITLE
Data format: TSV

### PR DESCRIPTION
**Pull request by joverlee521**
_[docs preview](https://nextstrain--238.org.readthedocs.build/en/238/reference/data-formats.html)_

## Description of proposed changes

Add TSV section to the Data Format docs, based on discussion in https://github.com/nextstrain/augur/issues/1566.

## Related issue(s)

Resolves https://github.com/nextstrain/docs.nextstrain.org/issues/237

## Checklist

- [x] Checks pass (failing rtd redirect unrelated to changes in this PR)

### CODE REVIEW

**Comment by joverlee521** [comment]
Hmm, both rtd redirects and docs build failed with 403 Client Error...maybe related to https://github.com/readthedocs/readthedocs.org/issues/11763.

**Comment by joverlee521** [comment]
Created [issue to track rtd redirect error](https://github.com/nextstrain/readthedocs-cli/issues/7), but that should not block this PR since I'm not making any changes to the redirect paths: https://github.com/nextstrain/docs.nextstrain.org/blob/a15d2ab7523d259cc1b4acdd631452e4402d5850/.github/workflows/sync-redirects.yaml#L7-L9

File: src/reference/data-formats.rst [hunk 1]
```diff
@@ -2,6 +2,37 @@
 Data formats
 ============
 
+.. contents:: Table of Contents
+   :local:
+
+TSV
+===
+
+Nextstrain generally uses TSV files for metadata.
```
[the remark this answers is not in the document]
Comment (genehack) [reply src/reference/data-formats.rst:11]: yeah, i like that! +1

**Review by genehack — APPROVED** [review]

**Comment by jameshadfield** [comment]
Is this the correct place to document suggested python code for reading/writing? Would be great to have snippets to reuse

**Comment by joverlee521** [comment]
> Is this the correct place to document suggested python code for reading/writing? Would be great to have snippets to reuse

Ah, that's true! I'll add it tomorrow.

**Pull request merged**